docs/next.config.js

```diff
@@ -14,14 +14,17 @@ module.exports = {
     DOCS_ROOT: __dirname,
     GESTALT_ROOT: root,
   },
-  webpack: (
-    config /*: WebpackConfig */,
-    { dev } /*: {| dev: boolean |} */,
-  ) /*: WebpackConfig */ => ({
-    ...config,
-    watchOptions: {
-      ...config.watchOptions,
-      poll: dev ? 500 : false,
-    },
-  }),
+  webpack: (config /*: WebpackConfig */, { dev } /*: {| dev: boolean |} */) /*: WebpackConfig */ =>
+    // These modules resolve by default in the server-side functions.
+    ({
+      resolve: {
+        // Explicitly tell webpack to ignore resolving "fs" for the client bundle
+        fallback: { fs: false, path: false },
```

Comment:
Answer here: https://github.com/vercel/next.js/discussions/12124

To get the markdown files from disk, we use the `fs` module. Since we're allowed to use it in the `gestStaticProps` method, we're okay. 

However, webpack also tries to add it to the client and causes an error. This line should prevent that.